doc/guides/security_release_process.md

```diff
@@ -2,118 +2,97 @@
+* [ ] Open an [issue](https://github.com/nodejs-private/node-private) titled
+  `Next Security Release`, and put this checklist in the description.
 
 * [ ] Get agreement on the list of vulnerabilities to be addressed:
-  * ***LINKS TO VULNS...***
+  * ***H1 REPORT LINK***: ***DESCRIPTION*** (***CVE or H1 CVE request link***)
+    * v10.x, v12.x: ***LINK to PR URL***
+  * ...
+
+* [ ] PR release announcements in [private](https://github.com/nodejs-private/nodejs.org-private):
+  * (Use previous PRs as templates, don't forget to update the site banner, and
+    the date in the slug so that it will move to the top of the blog list.)
+  * [ ] pre-release: ***LINK TO PR***
+  * [ ] post-release: ***LINK TO PR***
 
 * [ ] Get agreement on the planned date for the release: ***RELEASE DATE***
 
-* [ ] Validate that all vulnerabilities have been assigned a CVE. Upstream deps
-  such as OpenSSL and NPM will have CVEs, issues reported on H1 may have CVEs,
-  otherwise allocate them by following the
-  [cve_management_process](https://github.com/nodejs/node/blob/master/doc/guides/cve_management_process.md).
+* [ ] Get release team volunteers for all affected lines:
+  * v12.x: ***NAME of RELEASER(S)***
+  * ... other lines, if multiple releasers
 
-* [ ] Co-ordinate with the Release team members to line up one or more releasers
-  to do the releases on the agreed date. Releaser: ***NAME of RELEASER(S)***
+## Announcement (one week in advance of the planned release)
 
-* [ ] Prep for the security announcements by getting agreement on drafts (use
-  previously announced releases as the template):
-  * pre-release: ***LINK TO COMMENT ON THIS ISSUE CONTAINING DRAFT***
-  * post-release: ***LINK TO COMMENT ON THIS ISSUE CONTAINING DRAFT***
+* [ ] Check that all vulnerabilities are ready for release integration:
+  * PRs against all affected release lines or cherry-pick clean
+  * Approved
+  * Pass `make test`
+  * Have CVEs
+  * Described in the pre/post announcements
 
-## Announcement (one week in advance of the planned release)
+* [ ] Pre-release announcement [email][]: ***LINK TO EMAIL***
+  (Get access from existing manager: Ben Noordhuis, Rod Vagg, Michael Dawson)
 
-* [ ] Send pre-release announcement to
-  https://groups.google.com/forum/#!forum/nodejs-sec.
-  One of the existing managers can give access (Ben
-  Noordhuis, Rod Vagg, Michael Dawson). ***LINK TO EMAIL***
-
-* [ ] Post pre-release announcement in vulnerabilities section of Nodejs.org
-  blog (https://github.com/nodejs/nodejs.org/tree/master/locale/en/blog/vulnerability).
-  Use last pre-release announcement as a template (it includes blog metadata
-  such as updates to the banner on the Node.js website to indicate security
-  releases are coming).  Submit PR and land immediately. Text was already
-  reviewed in security repo.  ***LINK TO BLOG PR AND POST***
-
-* [ ] Open an issue in the build working repository with a notification of the
```

Comment:
This should stay? It didn’t happen for the recent security releases and the release ended up delayed due to infra issues.

Reply:
I don't mind putting it back in, its just one step of many. That said, I purposefully propose removing it. I'm trying to decrease the amount of personalized notifications that are required during release. The nodejs sec mailing list is close to 2 emails per release, one that a release is coming in a week, one when its done. I would like to request that anybody who has responsibilities to the release process should be subscribed to it. Anybody who didn't get  the email but does get nodejs/build notifications would have seen  https://github.com/nodejs/build/issues/2155 opened at the beginning of the release process (I didn't open another issue because I thought that was enough of a heads up that a release was in progress, and that more issues opened would just be nagging). I think timezone offsets and limited numbers of people who can solve specific infra problems are a bottleneck, but sufficient advertising of a release date is not.